### ISSUE TITLE
[MPMD-268] Missing warnings about deprecated rules

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-pmd-plugin#270]
**[Andreas Dangel](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=adangel)** opened **[MPMD-268](https://issues.apache.org/jira/browse/MPMD-268?redirect=false)** and commented

If you use a custom ruleset, which contains deprecated rules, then a warnings is expected.

E.g. referencing this rule

```
<rule ref="rulesets/java/basic.xml/CollapsibleIfStatements"/>
```

Should give the warning:

```
[WARNING] Use Rule name category/java/design.xml/CollapsibleIfStatements instead of the deprecated Rule name rulesets/java/basic.xml/CollapsibleIfStatements. PMD 7.0.0 will remove support for this deprecated Rule name usage.
```

 

---
No further details from [MPMD-268](https://issues.apache.org/jira/browse/MPMD-268?redirect=false)
